metricbeat/module/elasticsearch/index_recovery/_meta/fields.yml

```diff
@@ -38,6 +39,7 @@
       type: keyword
       description: >
         Source node id.
+    # ECS?
```

Comment:
Changed to `keyword` in https://github.com/elastic/beats/pull/10350 per discussion in comment thread above about a similar field.